### PR TITLE
[interactive_media_ads] Updates readme to clarify currently supported features

### PR DESCRIPTION
**Pull request by bparrishMines**
## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I [linked to at least one issue that this PR fixes] in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style], or this PR is [exempt from CHANGELOG changes].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/blob/master/docs/contributing/Chat.md
[linked to at least one issue that this PR fixes]: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md#overview
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#version
[following repository CHANGELOG style]: https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog-style
[exempt from CHANGELOG changes]: https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog
[test-exempt]: https://github.com/flutter/flutter/blob/master/docs/contributing/Tree-hygiene.md#tests

### CODE REVIEW

**Comment by bparrishMines** [comment]
cc @harold1208

File: packages/interactive_media_ads/README.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ a separate video player positioned on top of the app's content video player.
 |-------------|---------|-------|
 | **Support** | SDK 21+ | 12.0+ |
 
-**This package is still in development.**
+**NOTE:**
+* The initial release for this package supports linear pre-roll video ads on iOS and Android
+* platforms. Please view/report any issues [here](https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+interactive_media_ads%22).
```
Comment (stuartmorgan) [inline packages/interactive_media_ads/README.md:18]: What's the reason for the issue link? We have the metadata set that causes `pub.dev` to put an issues link in the sidebar, and that's where we expect clients to be seeing the README.

**Review by stuartmorgan — APPROVED** [review]
LGTM, although I would prefer we not have the issue link, per my comment below.

File: packages/interactive_media_ads/README.md (hunk 1 above)
Comment (bparrishMines) [reply packages/interactive_media_ads/README.md]: Ah, that's right. That was my suggestion. I forgot that is already included in the pubspec. I removed it from here. 

cc @harold1208 to confirm this is fine as well

Comment (harold1208) [reply packages/interactive_media_ads/README.md]: Confirming this is fine, thanks!

**Pull request merged**